crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -73,11 +58,25 @@ impl Extension for LoggerExtension {
         request: Request,
         next: NextPrepareRequest<'_>,
     ) -> ServerResult<Request> {
-        self.set_session_id(ctx.data_opt::<SocketAddr>().copied())
-            .await;
         next.run(ctx, request).await
     }
 
+    async fn resolve(
```

Comment:
Added this hook to get the path node in order to measure the top level queries that are being called.